client/src/components/Layout/SidebarItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+
+import styles from './SidebarItem.module.css';
+
+type Item = {
+  name: string;
+  url: string;
+};
+
+const SidebarItem: React.FC<Item> = (props) => {
+  const location = useLocation();
```

Comment:
No need to use `useLocation`. NavLink has built in way of detecting if link is active. Read the [docs](https://reactrouter.com/en/main/components/nav-link) and if any questions appear - hit me up